lib/solarwinds_apm/api/custom_instrumentation.rb

```diff
@@ -14,6 +14,37 @@ def self.included(base)
       end
 
       module ClassMethods
+        # Helper function to instrument custom function
+        #
+        # `add_tracer` can add a custom span to the specified instance or class method that is already defined.
+        # It requires the custom span name and optionally takes the span kind and additional attributes
+        # in hash format.
+        #
+        # === Argument:
+        #
+        # * +method_name+ - (String) A non-empty string that match the function name that need to be instrumented
+        # * +span_name+ - (String, optional, default = method_name) A non-empty string that define the span name (default to method_name)
+        # * +options+ - (Hash, optional, default = {}) A hash with desired attributes
+        #
+        # === Example:
```

Comment:
in this example there is no `include SolarWindsAPM::API::Tracer`, should there be?

Reply:
Yes, updated.